ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -30,13 +30,35 @@
     - name: Dump the logs of all pods
       ansible.builtin.shell: |
         all_pods=$(oc get pods -o name)
+        # this is implicitly relying on the fact that the pod name
+        # prefixed with "pod/" so that the redirect places the logs
+        # in the correct pod directory.
         mkdir pod
-        for pod in $all_pods; do
-          echo $pod
-          oc logs $pod > ${pod}-logs.txt
-          oc get -o yaml $pod > ${pod}.yaml
-          oc describe $pod > ${pod}-describe.txt
+        for pod in ${all_pods}; do
+          echo ${pod}
+          oc logs ${pod} > ${pod}-logs.txt
+          oc get -o yaml ${pod} > ${pod}.yaml
+          oc describe ${pod} > ${pod}-describe.txt
         done
+        # so we rename the directory to "pods" to consistently
+        # use plural for directories.
+        mv pod pods
+      args:
+        chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
+      changed_when: true
+      ignore_errors: true
+
+    - name: Dump all openstack CRs
+      ansible.builtin.shell: |
+        mkdir crs
+        pushd crs
+          all_crds=$(oc get crd | grep openstack | awk '{print $1}' | awk -F '.' '{print $1}')
```

Comment:
We can remove the `| grep openstack`:
`oc get crd | awk '/openstack/ { print $1}' | awk -F '.' '{print $1}'`
I'm pretty sure we may even get a one-call awk, but I'm not fluent enough. Reducing the amount of pipe is always a good thing :).

Reply:
ya i was debating about removing the grep and doing  awk '/openstack/ { print $1}' but i also didn't know how to collapse the two awks so i gave up on both.

ill fix this shortly